spec.json

```diff
@@ -0,0 +1,4 @@
+[{
```

Comment:
nit: we shouldn't add this file here - usually for unit tests, since we're using `NewMemMapFs` we can just do a string w/ the json like
```golang
const testJSON = `[{"instanceSize":"S30_LOWCPU_NVME", "nodeCount":2}]`
```
and then this file would be added in the e2e folders once you do the e2e.

not strong opinion so if you still prefer to keep the file i'd ask to at least move it to cli/atlas/searc/nodes and call it `test_spec.json` to make it clear is a test file.

Reply:
My bad, I checked this in by accident, this is my testing file

Reply:
Removed